editor/src/clj/editor/defold_project.clj

```diff
@@ -979,34 +1227,11 @@
   (let [pruned-evaluation-context (g/pruned-evaluation-context evaluation-context cache-entry-pred)]
     (g/update-cache-from-evaluation-context! pruned-evaluation-context)))
 
-(defn- log-cache-info! [cache message]
-  ;; Disabled during tests to minimize log spam.
-  (when-not (Boolean/getBoolean "defold.tests")
-    (let [{:keys [total retained unretained limit]} (g/cache-info cache)]
-      (log/info :message message
-                :total total
-                :retained retained
-                :unretained unretained
-                :limit limit))))
-
 (defn update-system-cache-build-targets! [evaluation-context]
-  (update-system-cache-from-pruned-evaluation-context! cached-build-target-output? evaluation-context)
-  (log-cache-info! (g/cache) "Cached build targets in system cache."))
+  (update-system-cache-from-pruned-evaluation-context! cached-build-target-output? evaluation-context))
 
 (defn update-system-cache-save-data! [evaluation-context]
-  (update-system-cache-from-pruned-evaluation-context! cached-save-data-output? evaluation-context)
-  (log-cache-info! (g/cache) "Cached save data in system cache."))
-
-(defn- cache-save-data! [project]
-  ;; Save data is required for the Search in Files feature, so we pull
-  ;; it in the background here to cache it.
-  (let [evaluation-context (g/make-evaluation-context)]
-    (future
-      (error-reporting/catch-all!
-        ;; TODO: Progress reporting.
-        (g/node-value project :save-data evaluation-context)
-        (ui/run-later
-          (update-system-cache-save-data! evaluation-context))))))
```

Comment:
No need for this anymore! We have the `save-value` outputs needed for search cached and ready immediately after loading.